src/libraries/System.Reflection.Emit/src/Resources/Strings.resx

```diff
@@ -171,4 +171,16 @@
   <data name="InvalidOperation_GenericParametersAlreadySet" xml:space="preserve">
     <value>The generic parameters are already defined on this MethodBuilder.</value>
   </data>
+  <data name="Argument_ShouldOnlySetVisibilityFlags" xml:space="preserve">
+    <value>Should only set visibility flags when creating EnumBuilder.</value>
+  </data>
+  <data name="Argument_ConstantDoesntMatch" xml:space="preserve">
+    <value>Constant does not match the defined type.</value>
+  </data>
+  <data name="Argument_ConstantNull" xml:space="preserve">
```

Comment:
I wonder if we need the differentiation here - the one above may be acceptable for null as well, plus considering there is only one case for the null check.

Reply:
Yes, the above acceptable, but it might be cleared than the above, also the exception messages and throwing logic copied from corelib, it might be better to be consistent with corelib messaging, let me know if you think otherwise.